### ISSUE TITLE
The `max-empty-lines` fix writes no run standing in front of a stray semicolon behind a rule, and the warning goes with it

### ISSUE THREAD

**Issue by firefoxic** [stylelint-stylistic/stylelint-stylistic#584]
### What minimal example or steps are needed to reproduce the bug?

A stray semicolon standing behind a rule is filed by PostCSS into that rule's `raws.ownSemicolon`, together with the whitespace in front of it, and the block's `raws.after` opens behind the semicolon. `max-empty-lines` counts the empty lines in front of such a semicolon and reports them, and its fix writes nothing over them — since #481 it writes the run behind the semicolon, which is the block's own raw, and nothing in front of it.

Measured with `@stylistic/max-empty-lines: 1` on `177f15b` and on the branch of #481:

| input | without `--fix` | with `--fix` on `177f15b` | with `--fix` on the branch of #481 |
| --- | --- | --- | --- |
| `a {⏎\tb {}⏎⏎⏎;⏎⏎⏎}` | `4:1`, `7:1` | unchanged, nothing reported | `a {⏎\tb {}⏎⏎⏎;⏎⏎}`, and `4:1` is reported on the next run |

`postcss.parse` of the input: `ownSemicolon` is `⏎⏎⏎;`, the block's `raws.after` is `⏎⏎⏎`.

### What minimal configuration is needed to reproduce the bug?

```json
{
	"plugins": ["@stylistic/stylelint-plugin"],
	"rules": {
		"@stylistic/max-empty-lines": 1
	}
}
```

### What did you expect to happen?

Both runs are collapsed to the option's count, or the warning about the first one stays.

### What actually happened?

The run in front of the semicolon stays as it was, and on `177f15b` the warning about it is reported as fixed.

### Notes

Found by the review of #481. The same semicolon moves a container's `source.end` past its brace and is the subject of #568; the sweep of #481 could not see this shape, since no last node of its corpus is a rule.
